.solhintignore

```diff
@@ -0,0 +1,12 @@
+# Doesn't need to lint dev files
+lib
+scripts
+test
```

Comment:
we don't lint scripts and tests?

Reply:
It format everything but doesn't lint tests and scripts as a lot of errors will come from these files but cannot be fixed for example console.log rule, naming conventions ...